pyaedt/edb_core/edb_data/components_data.py

```diff
@@ -895,27 +895,37 @@ def assign_s_param_model(self, file_path, name=None, reference_net=None):
         return self._set_model(model)
 
     @pyaedt_function_handler
-    def assign_rlc_model(self, res, ind, cap, is_parallel=False):
+    def assign_rlc_model(self, res=None, ind=None, cap=None, is_parallel=False):
```

Comment:
```suggestion
    def assign_rlc_model(self, res=0, ind=0, cap=0, is_parallel=False):
```

Reply:
0 can be a value. I need None to disable or enable

Reply:
@maxcapodi78 OK, got it. It makes sense to allow disabling the creation or R, L or C.